project/.travis/before_install_test.sh.twig

```diff
@@ -2,11 +2,12 @@
 set -ev
 
 if [ "${TRAVIS_PHP_VERSION}" != "hhvm" ]; then
-    mv "$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini" /tmp
-    echo "memory_limit=3072M" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+    PHP_INI_DIR="$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/"
+    TRAVIS_INI_FILE="$PHP_INI_DIR/travis.ini"
+    echo "memory_limit=3072M" >> "$TRAVIS_INI_FILE"
 
-    if [ ${TRAVIS_PHP_VERSION} '<' '7.0' ]; then
-        echo "extension=mongo.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+    if [ "$TRAVIS_PHP_VERSION" '<' '7.0' ]; then
+        echo "extension=mongo.so" >> "$TRAVIS_INI_FILE"
```

Comment:
There is no need for that if the string does not contain something other than the variable